public/components/integrations/components/add_integration_flyout.tsx

```diff
@@ -91,7 +91,7 @@ export const doPropertyValidation = (
 };
 
 export function AddIntegrationFlyout(props: IntegrationFlyoutProps) {
-  const { onClose, onCreate, integrationName, integrationType } = props;
+  const { onClose, onCreate, integrationName, integrationType, http } = props;
```

Comment:
note only. clients/services like http might be better to be stored as context value or somewhere global. passing through props can get messy since many things need it

Reply:
You can use this: https://github.com/opensearch-project/dashboards-observability/blob/main/public/framework/core_refs.ts

Reply:
Just switched to using coreRefs